cmd/satellite/main.go

```diff
@@ -26,6 +26,7 @@ import (
 // Satellite defines satellite configuration
 type Satellite struct {
 	Database string `help:"satellite database connection string" default:"sqlite3://$CONFDIR/master.db"`
+	Schema   string `help:"satellite database schema name" default:""`
```

Comment:
You should not need this. You can use connstr to pass it in.

Reply:
if i dont pass in the schema, then i need to parse it out of the connstr before I can call `db.CreateSchema`.  I thought passing this in was cleaner. thoughts?

Reply:
Do you need to explicitly create the schema? But, I would rather handle it transparently without an extra flag.

Reply:
I think I do need to create the schema explicitly. At least it was erroring before I added did. Ok let me try to change it to not need an extra flag.